### PR TITLE
move `prefixUtilityClasses` helper to Platform

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
I have a separate branch that uses this helper in some new code that lives in `platform/` so it feels like a good time to elevate this helper up out the `applications/` folder and into `platform/`.

## Testing done
Local + existing test

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/helpers.js [hunk 1]
```diff
@@ -76,20 +76,3 @@ export const transformServiceHistoryEntryIntoTableRow = entry => ({
     </>
   ),
 });
-
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/helpers.js:79]: this is the function that I moved.

File: src/platform/utilities/tests/react-hooks.unit.spec.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { expect } from 'chai';
 
-import { usePrevious } from './react-hooks';
```
Comment (erikphansen) [inline src/platform/utilities/tests/react-hooks.unit.spec.js:5]: I moved this unrelated test file into the `tests` dir, where it should have been to begin with.

**Review by sanlouise — APPROVED** [review]
LGTM! 👍

**Review by rianfowler — APPROVED** [review]
